task_netcore/task4/Client/src/modules/AccountForm/containers/AccountFormContainer.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { clearErrorMessage, axiosLogin, axiosRegister } from '../actions';
+import { validateUserName, validatePassword } from '../../../Validation';
+import AccountForm from '../views';
+import { withRouter } from 'react-router-dom';
+import { applicationRoutes, errorMessagesEnum } from '../../../Constants';
+
+class AccountFormContainer extends React.Component {
+  componentDidMount() {
+    this.props.clearErrorMessage();
+  }
+
+  handleSubmit = (values) => {
+    const user = {
+      userName: values.userName,
+      passWord: values.password
+    }; 
+
+    if (this.currentPath() === applicationRoutes.registerFormRoute) {
+      axiosRegister(this.props.dispatch, user, this.props.history);
+    }
+
+    if (this.currentPath() === applicationRoutes.loginFormRoute) {
+      axiosLogin(this.props.dispatch, user, this.props.history);
+    }
+  };
+
+  currentPath = () => {
```

Comment:
this should be called something like `getCurrentPath`. Always start function's name with a verb.